demos/src/scss/demo.scss

```diff
@@ -60,32 +61,34 @@ body {
 	text-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
 }
 
-@include oFtIconsFontFace();
-
 .demo__icon {
 	@include oFtIconsBaseIconStyles();
 	display: block;
 	font-size: 20px;
-	line-height: 30px;
+	line-height: 24px;
 	color: #ffffff;
 }
 
-.demo__icon__search {
+.demo__icon--search {
```

Comment:
If you're including o-icons via the build service in the demos, why not use them like that directly on the demo pages?